### PR TITLE
tests: showcase bug 7286 (tls) - v1

### PR DESCRIPTION
**Pull request by jufajardini**
This also exposes https://redmine.openinfosecfoundation.org/issues/7287.

Related to
Bug https://redmine.openinfosecfoundation.org/issues/7286

To accompany https://github.com/OISF/suricata/pull/11843

Expectation:
- tests should fail not only due to the reported bug, but also because `certificate` and `chain` were absent from `schema.json`

## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/7286

### CODE REVIEW

**Comment by satta** [comment]
Thank you! 👍🏻

**Comment by jufajardini** [comment]
> Thank you! 👍🏻

🙇🏽

File: tests/tls/bug-7286-tls-metadata-02/test.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+requires:
+    min-version: 7.0.6
```
Comment (inashivb) [inline tests/tls/bug-7286-tls-metadata-02/test.yaml:2]: This looks good but sadly due to the json schema issue, the test won't work even on this min version..
Wondering if there should be an option to disable schema checks for tests like these..

Comment (jufajardini) [reply tests/tls/bug-7286-tls-metadata-02/test.yaml:2]: Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.

Should then maybe this one be merged first? https://github.com/OISF/suricata/pull/11845

Comment (inashivb) [reply tests/tls/bug-7286-tls-metadata-02/test.yaml:2]: > Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.
> 

I think it'll have to be a test.yaml option as validation is run by s-v and not the CI checks.

> Should then maybe this one be merged first? [OISF/suricata#11845](https://github.com/OISF/suricata/pull/11845) (or at least the Schema fixing PR)

We should indeed merge the Suricata PRs and perhaps this one too.

**Review by catenacyber — CHANGES_REQUESTED** [review]
Would you rebase and have green CI for main7 ?

**Comment by jufajardini** [comment]
> Would you rebase and have green CI for main7 ?

on it.

**Comment by jufajardini** [comment]
Follow up: https://github.com/OISF/suricata-verify/pull/2073

File: tests/tls/bug-7286-tls-metadata-02/test.yaml (hunk 1 above)
Comment (jufajardini) [reply tests/tls/bug-7286-tls-metadata-02/test.yaml:2]: > > Maybe a PR template option, or label? Like, we still want the test to be complete, and hopefully update the schema as SV highlights an issue like this, but then at least we'd still be able to check whether the tests are testing what they're supposed to, in the meanwhile.
> 
> I think it'll have to be a test.yaml option as validation is run by s-v and not the CI checks.

Should I create a ticket?

Comment (inashivb) [reply tests/tls/bug-7286-tls-metadata-02/test.yaml:2]: No, it's trivial really and does not need to be tracked. Creating PR

**Pull request closed without merging**